Makefile

```diff
@@ -8,26 +8,33 @@ SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+SHELL := bash
+
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 PROJECTS = numpy scipy matplotlib pandas scikit-learn scikit-image networkx astropy sunpy
 ISSUE_DATA = $(patsubst %, devstats-data/%_issues.json, $(PROJECTS))
-PR_DATA = $(patsubst %, devstats-data/%_prs.json, $(PROJECTS))
+PR_DATA = $(patsubst %, devstats-data/%_PRs.json, $(PROJECTS))
 REPORTS = $(patsubst %, _generated/%/index.md, $(PROJECTS))
 
 $(REPORTS) : $(ISSUE_DATA) $(PR_DATA)
 
-devstats-data/%_issues.json :
-	devstats query -o devstats-data $* $*
+devstats-data :
+	mkdir devstats-data
+
+devstats-data/%_issues.json : | devstats-data
+	(cd devstats-data && \
+	export URL=$(shell curl -s https://api.github.com/repos/scientific-python/devstats-data/releases/latest | jq -r '.assets[] | select(.name | contains ("devstats-$*.")) | .browser_download_url') && \
```

Comment:
Non-functional comment as it doesn't affect functionality just cleans up this shell command; but I wonder if there is an easier way to pull this data, e.g. a shallow clone?

Reply:
Note that this downloads from the *release* bundle; it's not stored in the repo:

https://github.com/scientific-python/devstats-data/releases